sceptre/scipool/config/strides/sc-tag-options.yaml

```diff
@@ -13,6 +13,7 @@ dependencies:
 sceptre_user_data:
   Departments: !file sc-tag-options/internal/Departments.json
   Projects: !file sc-tag-options/internal/Projects.json
+  CostCenters: !file https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.3.6/tags/SageFinancialProgramCodes.json
```

Comment:
STRIDES products (like BMGF-Ki products) should not ask for a Sage Cost Center.

Reply:
fixed